langgraph/src/pregel/read.ts

```diff
@@ -119,7 +120,7 @@ export class PregelNode<
 > extends RunnableBinding<RunInput, RunOutput, RunnableConfig> {
   lc_graph_name = "PregelNode";
 
-  channels: Record<string, string> | string;
+  channels: Record<string, string> | string[];
```

Comment:
This type changes requires updating the implementation of `_prepareNextTasks()` and a bunch of tests.